src/main/java/com/xebialabs/overthere/local/LocalConnection.java

```diff
@@ -129,7 +124,11 @@ public OverthereProcess startProcess(CmdLine cmd) {
 
         try {
             logger.debug("Creating " + os + " process with command line [{}]", obfuscatedCmd);
-            final ProcessBuilder pb = new ProcessBuilder(cmd.toCommandArray(os, false));
+            String[] args = cmd.toCommandArray(os, false);
```

Comment:
don't `toCommandArray` here, but check with the if, and pass the `CmdLine` as an argument to the method, and return the new `CmdLine` from it.